osbuild/buildroot.py

```diff
@@ -220,6 +220,10 @@ def run(self, argv, monitor, timeout=None, binds=None, readonly_binds=None, extr
         mounts += ["--tmpfs", "/tmp"]
         mounts += ["--bind", self.var, "/var"]
 
+        # Create a usable /var/tmp, see bib issue#223
```

Comment:
Could you link this? A casual reader likely doesn't know what `bib` is.

Reply:
Good point, thank you! Updated